job-server/src/spark.jobserver/io/JobDAO.scala

```diff
@@ -19,7 +19,11 @@ case class JobInfo(jobId: String, contextName: String,
   def isRunning: Boolean = !endTime.isDefined
   def isErroredOut: Boolean = endTime.isDefined && error.isDefined
 }
-
+object JobInfo {
```

Comment:
I think JobStatus will be better name.

That way you can avoid STATUS prefix too and have something like this

JobStatus {

val Running = "RUNNING"
val Error = "ERROR"
val Finished = "FINISHED"

}